src/test/scala/uk/gov/nationalarchives/signcookies/LambdaTest.scala

```diff
@@ -66,4 +66,14 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
 
     decode[LambdaResponse](byteArrayCaptor.getValue.map(_.toChar).mkString).toOption.get.statusCode should equal(200)
   }
+
+  "the lambda" should "does not error if the origin header has an upper case first letter" in {
```

Comment:
Slightly reword (delete 'does'): "not error if the ...

Reply:
Me fail English, that's unpossible.